ivy.el

```diff
@@ -5115,7 +5115,7 @@ EVENT gives the mouse position."
   (ivy--occur-press-update-window)
   (when (save-excursion
           (beginning-of-line)
-          (looking-at "\\(?:./\\|    \\)\\(.*\\)$"))
+          (looking-at "\\(?:./\\|.\\\\\\|    \\)\\(.*\\)$"))
```

Comment:
Why not a character class instead:
```el
"\\(?:.[/\\]\\|    \\)\\(.*\\)$"
```
And shouldn't that `.` be `\.` instead?
```el
"\\(?:\\.[/\\]\\|    \\)\\(.*\\)$"
```

Reply:
Well, I'm not sure about `.` being `\.` or not since it has been there from begining.